package com.dbase;

import java.sql.SQLException;

public class Runner {

    public static void main(String[] args) throws SQLException {
        DbUserStorage dbu = new DbUserStorage();

//        dbu.updateNameById(17, "Updated Name");

//        dbu.updatePasswordById(17, "1122");

//        System.out.println(dbu.getByUsername("oIfUi65").toString());

//        dbu.deleteById(14);

//        System.out.println(dbu.getAll().toString());
//        System.out.println(dbu.getAllByName("cQewX").toString());
//        System.out.println(dbu.getAllByAddress("OWKTN").toString());

//        User user = dbu.getByUsername("WatJd36").get();
//        System.out.println(dbu.exist(user));
//
//        System.out.println(dbu.existByID(21));

    }
}